build/utils/test_helpers.go

```diff
@@ -0,0 +1,33 @@
+package utils
+
```

Comment:
Let's rename this file to ```test_helper_test.go``` so that it doesn't get packed into the go binary.

Reply:
This file contains utils that are being used by other `_tests.go` files.

Reply:
@Or-Geva

This change is required for excluding this file from the final executable:

> When compiling packages, build ignores files that end in '_test.go'. 

https://pkg.go.dev/cmd/go@master#hdr-Compile_packages_and_dependencies

Reply:
Moved to testdata. Thanks @yahavi for the advice!